src/databricks/labs/ucx/workspace_access/manager.py

```diff
@@ -46,7 +46,13 @@ def factory(
             generic.Listing(generic.experiments_listing(ws), "experiment_id", "experiments"),
             generic.Listing(generic.models_listing(ws), "id", "registered-models"),
             generic.Listing(generic.tokens_and_passwords, "object_id", "authorization"),
-            generic.WorkspaceListing(ws, num_threads=num_threads, start_path=workspace_start_path),
+            generic.WorkspaceListing(
+                ws,
+                sql_backend=sql_backend,
+                inventory_database=inventory_database,
+                num_threads=num_threads,
+                start_path=workspace_start_path,
```

Comment:
Start path and num threads only have the effect in runtime.py, add them there and not here

Reply:
This is also invoked by GroupMigrationToolkit, shouldn't the num_thread and start_path used by that call to set?